Added organisation README page

luccam-scr opened the pull request:
Added organisation README page with introduction to Scramjet Framework and Scramjet Transform Hub

Commit:
Added organisation README page

MichalCz reviewed (approved):
A couple of minor comments - otherwise cool.

luccam-scr committed:
Implemented PR comments

Updted company info, Transform Hub moved to first place before Scramjet Framework
